src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -86,6 +88,21 @@ private function addValueField(FormInterface $form, AttributeInterface $attribut
     {
         $options = ['auto_initialize' => false, 'label' => $attribute->getName()];
 
+        if (OptionListAttributeType::TYPE === $attribute->getType()) {
+            $configuration = $attribute->getConfiguration();
+
+            if (OptionListAttributeConfigurationType::FORMAT_MULTIPLE_SELECTION === $configuration['format']) {
+                $options['multiple'] = true;
+            }
```

Comment:
maybe?:

```
else {
    if (count($attributeValue->getValue()) > 1) {
        $attributeValue->setValue($attributeValue->getValue()[0];
    }
}
```

2 issues to resolve:
* saving an empty option list value
* what happens when you change from a multi-select with multiple values back to a single dropdown?